SaintCoinach/Definitions/MJIName.json

```diff
@@ -1,4 +1,32 @@
 {
   "sheet": "MJIName",
-  "definitions": []
+  "defaultColumn": "Name",
+  "definitions": [
+    {
+      "name": "Singular"
+    },
+    {
+      "index": 1,
+      "name": "Adjective"
+    },
+    {
+      "index": 2,
+      "name": "Plural"
+    },
+    {
+      "index": 3,
+      "name": "PossessivePronoun"
+    },
+    {
+      "index": 4,
+      "name": "StartsWithVowel"
+    },
+    {
+      "index": 6,
+      "name": "Pronoun"
+    },
+    {
+      "index": 7,
+      "name": "Article"
+    }
```

Comment:
the ] is missing here and SaintCoinachCMD crashes when opening (currently not able to create a PR) @IcarusTwine

Reply:
I hotfixed on master branch